web/yo/app/scripts/directives/curationQueue.js

```diff
@@ -277,98 +275,42 @@ angular.module('oncokbApp')
-                            });
-                            if (_articles.length > 0) {
-                                generateEmail(_email, _curator, user.name, _articles, new Date().getTime())
-                                    .then(function() {
-                                        $scope.email.returnMessage += _curator + ' has been notified' + '.<br/>';
-                                        sendIndividualEmail(++curatorIndex);
-                                    }, function(error) {
-                                        $scope.email.returnMessage += '<span style="color: red;">Failed to send Email. Please mention following message to developer: ' + error + '.</span><br/>';
-                                        sendIndividualEmail(++curatorIndex);
-                                    });
-                            }
-                        } else {
-                            $scope.email.returnMessage += '<span style="color: red;">Can not find email for ' + _curator + '.</span><br/>';
-                            sendIndividualEmail(++curatorIndex);
-                        }
-                    } else {
-                        $scope.email.status.sending = false;
-                        $scope.data.curatorsToNotify = [];
                     }
-                }
-
-                function generateEmail(email, curatorName, adminName, articles, time) {
-                    var deferred = $q.defer();
-                    var content = 'Dear ' + curatorName.split(' ')[0] + ',\n\n';
-                    content += adminName + ' of OncoKB would like you curate the following publications in the indicated alteration, tumor type and section:\n\n';
-                    _.each(articles, function(article, index) {
-                        var tempArr = [index + 1 + ')', article.article];
-                        if (article.link) {
-                            tempArr = tempArr.concat(['(', article.link, ')']);
-                        }
-                        if (article.variant) {
-                            tempArr = tempArr.concat(['Alteration:', article.variant + ',']);
-                        }
-                        if (article.tumorType) {
-                            tempArr = tempArr.concat(['Tumor type:', article.tumorType + ',']);
-                        }
-                        if (article.section) {
-                            tempArr = tempArr.concat(['Section:', article.section]);
-                        }
-                        content += tempArr.join(' ') + '\n';
-                    });
-                    content += '\nPlease try to curate this literature within two weeks (' + new Date(time + 12096e5).toDateString() + ') and remember to log your hours for curating this data.\n\n';
-                    content += 'If you have any questions or concerns please email or slack ' + adminName + '.\n\n';
+                    var content = 'Dear ' + item.get('curator').split(' ')[0] + ',\n\n';
```

Comment:
If you mean email content, it is not possible to be empty. Since you need to input at least article before you can add it.